cli/object_command.go

```diff
@@ -82,7 +82,7 @@ NOTE: This is an experimental feature.
 	put.Flag("name", "Override the name supplied to the object store").StringVar(&c.overrideName)
 	put.Flag("description", "Sets an optional description for the object").StringVar(&c.description)
 	put.Flag("header", "Adds headers to the object").Short('H').StringsVar(&c.hdrs)
-	put.Flag("progress", "Disables progress bars").Default("true").BoolVar(&c.noProgress)
+	put.Flag("no-progress", "Disable progress bars").UnNegatableBoolVar(&c.noProgress)
```

Comment:
since the new cli library now have negatable booleans I think I'd rather like:

```
put.Flag("progress", "Disable progress bars").Default("true").BoolVar(&c.progress)
```

like in stream backup and restore, its a bit more in tune also with how we handle other boolean flags now

Reply:
OK. Update coming!